tsconfig.base.json

```diff
@@ -0,0 +1,15 @@
+{
+  "include": ["src/**/*", "types/**/*"],
+  "compilerOptions": {
+    "moduleResolution": "node",
+    "declaration": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "resolveJsonModule": true,
+    "strict": true,
+    "stripInternal": true,
+    "forceConsistentCasingInFileNames": true,
```

Comment:
If you'd like to have more checks, e.g. the same level as our Angular 13 front-ends, you can consider adding these flags:
```json
    "noImplicitOverride": true,
    "noPropertyAccessFromIndexSignature": true,
    "noImplicitReturns": true,
    "noFallthroughCasesInSwitch": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
```